code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/SmallImageUIState.kt

```diff
@@ -0,0 +1,10 @@
+package com.adobe.marketing.mobile.aepcomposeui.aepui.state
+
+/**
+ * Data class representing the state of a small image UI.
+ *
+ * @property title The title of the small image UI.
+ * @property dismissed Indicates whether the UI has been dismissed.
+ */
+data class SmallImageUIState(val title: String?, val dismissed: Boolean = false) : AEPUIState
```

Comment:
Wondering if we want to explore a top level _AepCardState_ interface that holds different behaviors like dismissed, selected etc.
```
  interface AepCardState {
    val dismissed: Boolean,
    val selected:  Boolean
}

data class SmallImageUIState(override var dismissed: Boolean = ...) : AepCardState

```